projects/angular/src/layout/_card.clarity.scss

```
@@ -51,11 +52,11 @@
     display: block;
     background-color: card-variables.$clr-card-bg-color;
     width: 100%;
-    margin-top: tokens.$cds-global-space-9;
+    margin-top: density.$clr-base-vertical-offset-2xl;
     @extend %card-border-appearance;
 
     .alert {
-      margin: 0 0 tokens.$cds-global-space-5 0;
+      margin: 0 0 density.$clr-base-vertical-offset-m 0;
```

Make this rule to use `%card-space-between;`

```suggestion
      @extend %card-space-between;;
```

---

Fixed in [e58df88](https://github.com/vmware-clarity/ng-clarity/pull/1896/commits/e58df88b083135a2b67d6ae4b3bfe0d091a12d57)